### ISSUE TITLE
Failing test verifying if character newlines are preserved for Preformatted block 

### ISSUE THREAD

**Issue by gziolo** [WordPress/gutenberg#14014]
**Describe the bug**
I observe the following failing e2e test on Travis quite often since this morning.

```
AIL packages/e2e-tests/specs/blocks/preformatted.test.js
  ● Preformatted › should preserve character newlines
    No node found for selector: button[aria-label="More options"]
      at assert (../../node_modules/puppeteer/lib/helper.js:282:11)
      at Frame.click (../../node_modules/puppeteer/lib/FrameManager.js:590:5)
```
Example of the failed job on Travis for the lastest PR merged into `master:
https://travis-ci.org/WordPress/gutenberg/jobs/496540012

The test was added in #13799 by @iseulde yesterday.

**To Reproduce**
`npm run test-e2e` on Travis.

**Expected behavior**
Test passes all the time.

**Comment by youknowriad** [comment]
I may have fixed this instability in #13088

**Comment by iseulde** [comment]
Fixed in #13088. Thanks!
